### PR TITLE
Update README for 2021

### PR DESCRIPTION
**Pull request by scottdangelo**
Update README and screen shots.
Also, remove deprecated Cloud Foundry deployment.
Remove vestiges of K8s deployment.
Remove outdated test/ directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 # Monitor Custom Machine Learning engine with Watson OpenScale
 
-In this Code Pattern, we will log the payload for a model deployed on custom model serving engine using Watson OpenScale python sdk. We'll use [Keras to build a deep learning REST API](https://blog.keras.io/building-a-simple-keras-deep-learning-rest-api.html) and then monitor with [Watson OpenScale](https://www.ibm.com/cloud/watson-openscale/)
+In this Code Patten, we'll demonstrate how to monitor a machine learning model using [Watson OpenScale](https://www.ibm.com/cloud/watson-openscale/). You can using any model deployment serving engine, but we will create one using a Python Flask wrapper with Watson Machine Learning to simulate this.
```
Comment (rhagarty) [inline README.md]: typo - "patten"

File: README.md [hunk 2]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
 
-Create the following services:
+<details><summary>Setup on a VM</summary>
 
-* [Watson OpenScale](https://cloud.ibm.com/catalog/services/ai-openscale)
-  You will get the Watson OpenScale instance GUID when you run the notebook using the [IBM Cloud CLI](https://cloud.ibm.com/catalog/services/ai-openscale)
+### Custom Machine Learning Provider Setup for VM
 
-* [Databases for PostgreSQL DB](https://cloud.ibm.com/catalog/services/databases-for-postgresql)
+The code in flask.py can be used to start a gunicorn/flask application that can be hosted in a VM, such that it can be accessable from CPD system.
```
Comment (rhagarty) [inline README.md]: I think gunicorn should be capitalized. And maybe flask too?

I think "from your CPD system" > "from CPD system"

File: README.md [hunk 3]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+This code does the following:
+* It wraps a Watson Machine Learning model that is deployed to a space.
+* The hosting application URL should contain the SPACE ID and the DEPLOYMENT ID. The app can be used to talk to the target WML model/deployment.
+* Having said that, this is only for this tutorial purpose, and you can define your Custom ML provider endpoint in any fashion you want, such that it wraps your own custom ML engine.
+* The scoring request and response payload should confirm to the schema as described here at: https://dataplatform.cloud.ibm.com/docs/content/wsj/model/wos-frameworks-custom.html
```
Comment (rhagarty) [inline README.md]: "should confirm to the schema" - not sure what this means

File: README.md [hunk 4]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+or 
 
-### 4a. Deploy to IBM Cloud
+<details><summary>Run on RedHat OpenShift</summary>
 
-[![Deploy to IBM Cloud](https://cloud.ibm.com/devops/setup/deploy/button.png)](https://cloud.ibm.com/devops/setup/deploy?repository=https://github.com/IBM/monitor-custom-ml-engine-with-watson-openscale)
+## Prerequisites
 
-Click the ``Deploy to IBM Cloud`` button and hit ``Create`` and then jump to step 5.
+You will need a running OpenShift cluster, or OKD cluster. You can provision [OpenShift on the IBM Cloud](https://cloud.ibm.com/kubernetes/catalog/openshiftcluster).
```
Comment (rhagarty) [inline README.md]: OKD? should that be spelled out the first time?

File: README.md [hunk 5]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+1. [Create the config map](#2-create-the-config-map)
+1. [Get a secure endpoint](#3-get-a-secure-endpoint)
+1. [Run the web app](#4-run-the-web-app)
 
-> NOTE: Running locally will require Python 3.5 or 3.6 (later versions will not work with Tensorflow).
-If you run the server locally, it may not have a publicly addressible IP address, if you are behind a firewall or local router. You would therefore also have to run the [Jupyter notebook](https://jupyter.org/) locally as well.
+### 1. Create an OpenShift project
 
-* It is recommended that you use a [Python virtualenv](https://pypi.org/project/virtualenv/)
+* Using the OpenShift web console, select the `Application Console` view.
```
Comment (rhagarty) [inline README.md:85]: what version of OpenShift is this? Sure these screens still exist?

File: README.md [hunk 6]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
-```
+* Save the URL for the Route. You will use this in the [configuration notebook](#5-create-a-notebook-on-cloud-pak-for-data) in the section `1.2 Configure credentials` for the variable *CUSTOM_ML_PROVIDER_SCORING_URL*.
 
-### Run the [score_credit.py](test/score_credit.py) script following the  [test/README.md](test/README.md) instructions
+[![return](https://raw.githubusercontent.com/IBM/pattern-utils/master/deploy-buttons/return.png)](../../README.md#create-a-watson-openscale-service)
```
Comment (rhagarty) [inline README.md]: Don't need this return button. You never left the README.md page

File: README.md [hunk 7]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
 
-Navigate a browser to `http://<ip_address>:<port>/v1/deployments`
-or run the [test_deployments.py](test/deployments_api.py) script following the  [test/README.md](test/README.md) instructions.
+* From the OpenShift or OKD UI, under `Applications` ▷ `Routes` you will see your app.
+  * Click on the application `Name`.
+  * Under `TLS Settings`, click on `Edit`.
+  * Under `Security`, check the box for `Secure route`.
+  * Hit `Save`.
 
+## 4. Run the web app
```
Comment (rhagarty) [inline README.md]: When this section is expanded, the numbering is very confusing. You will see #4, then #3, then #4 again. Maybe make this a separate page? Or somehow indent the expand/collapse section more.

File: README.md [hunk 8]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+
+* In the Cloud Pak for Data instance, go the (☰) menu and under `Services` section, click on the `Instances` menu option.
+
+  ![Service](doc/source/images/services.png)
+
+* Find the `OpenScale-default` instance from the instances table and click the three vertical dots to open the action menu, then click on the `Open` option.
+
+  ![Openscale Tile](doc/source/images/services-wos-instance.png)
+
+* If you need to give other users access to the OpenScale instance, go the (☰) menu and under `Services` section, click on the `Instances` menu option.
```
Comment (rhagarty) [inline README.md]: "go to the ... "

File: README.md [hunk 9]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+
+* In your [IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage)  instance, create a bucket with a globally unique name. The UI will let you know if there is a naming conflict. This will be used in cell *1.3.1* as *BUCKET_NAME*.
+
+* In your [IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) instance, get the Service Credentials for use as `COS_API_KEY_ID`, `COS_RESOURCE_CRN`, and `COS_ENDPOINT`:
+
+  ![COS credentials](doc/source/images/cos-credentials.png)
+
+## 5. Create a notebook on Cloud Pak for Data
+
+* In your on-premise Cloud Pak for Data, click `New Project +` under Projects or, at the top of the page click `+ New` and choose the tile for `Data Science` and then `Create Project`.
```
Comment (rhagarty) [inline README.md:176]: does this really require on-premise CPD?

File: README.md [hunk 10]
```diff
@@ -19,141 +20,189 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+
+* Using the project you've created, click on `+ Add to project` and then choose the  `Notebook` tile, OR in the `Assets` tab under `Notebooks` choose `+ New notebook` to create a notebook.
```
Comment (rhagarty) [inline README.md]: extra space before Notebook

**Review by rhagarty — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: fixed.

File: README.md (hunk 3 above)
Comment (scottdangelo) [reply README.md]: Should be "conform"

File: README.md (hunk 4 above)
Comment (scottdangelo) [reply README.md]: Actually, I'd like to narrow it to OpenShift only, since the instructions are specifc to that.

File: README.md (hunk 5 above)
Comment (scottdangelo) [reply README.md:85]: Not sure. This image is from Nov 1 (you added). I haven't updated.

File: README.md (hunk 6 above)
Comment (scottdangelo) [reply README.md]: fixed

File: README.md (hunk 7 above)
Comment (scottdangelo) [reply README.md]: I agree. I've removed the numbers.

File: README.md (hunk 8 above)
Comment (scottdangelo) [reply README.md]: fixed

File: README.md (hunk 9 above)
Comment (scottdangelo) [reply README.md:176]: Yes, For these instructions. This is the supported notebook from the OpenScale team, so I want to use it. I don't think this pattern has had much use, so I don't think it's worth the effort to make it more universal.

File: README.md (hunk 10 above)
Comment (scottdangelo) [reply README.md]: fixed

**Comment by scottdangelo** [comment]
@rhagarty Should be all fixed.

**Review by rhagarty — APPROVED** [review]

**Pull request merged**